js/stores/appStore.js

```diff
@@ -129,14 +129,32 @@ function handleChangeSettingAction (settingKey, settingValue) {
     case settings.DEFAULT_ZOOM_LEVEL:
       filtering.setDefaultZoomLevel(settingValue)
       break
-    case settings.TOOLBAR_UI_SCALE: {
-      const newZoomLevel = zoomLevel[settingValue] || 0
-      BrowserWindow.getAllWindows().forEach(function (wnd) {
-        wnd.webContents.setZoomLevel(newZoomLevel)
-      })
-    } break
-    default:
+    case settings.TOOLBAR_UI_SCALE:
+      {
+        const newZoomLevel = zoomLevel[settingValue] || 0
+        BrowserWindow.getAllWindows().forEach(function (wnd) {
+          wnd.webContents.setZoomLevel(newZoomLevel)
+        })
+        break
+      }
+    case settings.HOMEPAGE:
+      {
+        let homeArray = settingValue.split('|')
+        homeArray = homeArray.map(page => {
+          page = page && page.trim()
```

Comment:
any reasons for why not just using `page = page.trim()`?

Reply:
yeah I can remove this one, it was copied from another place

Reply:
Done